src/App.jsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable react/jsx-props-no-spreading */
+
+import { React, useState, useMemo } from 'react';
+import { Routes, Route } from 'react-router-dom';
+
+import { useForm, FormProvider } from 'react-hook-form';
+
+import Home from './pages/Home';
+import Context from './Context';
+import Footer from './components/footer/Footer';
+import About from './pages/About';
+import Blog from './pages/Blog';
+import Contact from './pages/Contact';
+import Navbar from './components/navbar/Navbar';
+import CardPage from './pages/CardPage';
+
+function App() {
+  const [currentPage, setCurrenPage] = useState(1);
+  const [cardsPerPage] = useState(3);
```

Comment:
No sense in having a state without using update function, move to const outside form component body